tex/report.tex

```diff
@@ -134,8 +134,11 @@ \section{Methodology}\label{sec:method}
 Python, Ruby, and Bash, which do exactly the following:
 \begin{itemize}
     \item Fetch open repositories from GitHub, which have \ff{java} language
-    tag, have reasonably big but not too big number of stars, and are
-    of certain minimum size;
+    tag, have reasonably big but not too big number of stars, have either MIT or Apache License,
```

Comment:
@h1alexbel would be better to use a separate `\item` for the filtering by license

Reply:
@yegor256 fixed